dashboards/opensearch/opensearch_nodes.sp

```diff
@@ -21,3 +21,28 @@ node "opensearch_domain" {
 
   param "opensearch_arns" {}
 }
+
+
+node "opensearch_domain_arn" {
+  category = category.opensearch_domain
+
+  sql = <<-EOQ
+    select
+      arn as id,
+      title as title,
+      jsonb_build_object(
+        'ARN', arn,
+        'Domain ID', domain_id,
+        'Domain Name', domain_name,
+        'Engine Version', engine_version,
+        'Account ID', account_id,
+        'Region', region
+      ) as properties
+    from
+      aws_opensearch_domain
+    where
+      arn = any($1);
+  EOQ
+
+  param "opensearch_arns" {}
+}
```

Comment:
This node(opensearch_domain_arn) is not required we can use the existing node(opensearch_domain), we just need to correct the where clause. Can you please update node `opensearch_domain` as below and remove node `opensearch_domain_arn`
```
node "opensearch_domain" {
  category = category.opensearch_domain
  sql = <<-EOQ
    select
      arn as id,
      title as title,
      jsonb_build_object(
        'ARN', arn,
        'Domain ID', domain_id,
        'Domain Name', domain_name,
        'Engine Version', engine_version,
        'Account ID', account_id,
        'Region', region
      ) as properties
    from
      aws_opensearch_domain
    where
      arn = any($1);
  EOQ

  param "opensearch_arns" {}
}

```